framework/decode/vulkan_virtual_swapchain.cpp

```diff
@@ -363,7 +365,17 @@ VkResult VulkanVirtualSwapchain::CreateSwapchainResourceData(const DeviceInfo*
             }
         }
     }
-
+    // Intercept: Issue #1454
```

Comment:
Is this the only function that operates on `virtual_swapchain_images`?  I imagine you'd want special handling in any function that creates/updates/destroys that member.  And probably you want to not create them if you don't need them.

Reply:
Also `VulkanVirtualSwapchain::QueuePresentKHR()` skips the copy if the swapchain has been determined to have the bypass feature.

Reply:
I think the short answer is that once this mode is identified, every subsequent member of this class should behave like VulkanCapturedSwapchain.  It may be that isn't possible, because some calls to VulkanCapturedSwapchain members are made before the bypass can be identified.  An example is `ProcessSetSwapchainImageStatePreAcquire` may need to be implemented in `VulkanVirtualSwapchain`